meetings/2022-03-14.md

```diff
@@ -0,0 +1,108 @@
+* NAS: Currently it is possible, when you log user in based on WebID you know the WebID and discover OP from there. Then you can get OP keys. Is it possible that you can get token which was spoofed? There is no strong relationship between OP and the key. This is a strong change compared to Solid-OIDC. That may warrant to have it separate. 
+* AC: Let's say you have 2 storage servers A and B. With A there is this pre-existing trust where it trusts issuer P. Server B does not have the same trust with issuer P. What happens if a user tries to log in to server A with issuer P, they can do it fine. They would fail to log in to issuer B since they don't have that trust.
+* ...: What might be interesting as an alternative, is rather than trying to say I trust this particular issuer or specific issuers. I trust a [federation of issuer](https://openid.net/specs/openid-connect-federation-1_0.html). If you trust federation of let's say 20 or 30. It will not give away that much information. This may answer some of the use cases. We would have the question how you discover the federation.
+* MB: The federation would still be a group of issuers adhering to certain set of values. It may be more private, but still disclose some association with the values carried by such federation.
+* eP: Aaron, is it related to issuer restriction case? For example `acp:issuer`
+* AC: The first thing is AS looks up the WebID profile to check the issuer is trusted by this WebID profile. Because I can write a valid ID TOken claiming to be anyone. How can an AS know that this particular ID Token is to be trusted.
+* eP: We discussed having a public key that could proove the token was issued by someone trusted.
+* AC: We would have to be careful about token rotation.
+* eP: Yes, with per user key it's probably less frequent need, but it would indeed put some management burden on the user since they would have to do it.
+* MB: If this scenario is one that we would support, yes the user would need to manage key rotation. 
+* eP: If you want to avoid rotating that, it's a kind of trade-off. There could be a standard period for the user to update their profile. Some services prompt a user to change their password regularly, it seems reasonable.
+* MB: Issuer restrictions would get more complex here. If issuer is in charge of managing key pairs it could be a piece of software on your machine.
+* eP: Do you refer to self-issued OIDC like flows?
+* MB: Yes, it is de-facto the same kind of thing (or a possible implementation thereof).
+* eP: How to follow up on this issue/conversation? Is anyone interested in proposing initial text of how to address it?
+* MB: It might be interesting to involve people behind WebID-TLS. The main problem was usability issue, which was never resolved on the browser level. If that could be resolved server side.
```

Comment:
```suggestion
* MB: It might be interesting to involve people behind WebID-TLS. The main problem there was a usability issue, which was never resolved on the browser level. If that could be resolved server side, maybe WebID-TLS could be a good candidate?
```